models/balancer/balancer_schema.yml

```diff
@@ -172,4 +172,38 @@ models:
       - name: contract_address
       - name: median_price
    
-
+  - name: balancer_v2_liquidity
+    meta:
+      blockchain: ethereum, arbitrum, optimism, polygon, gnosis
+      project: balancer_v2
+      contributors: viniabussafi
+    config:
+      tags: ['balancer', 'amm', 'dex', 'liquidity', 'ethereum', 'arbitrum', 'optimism', 'polygon']
+    description: >
+      Liquidity by token on Balancer, an automated portfolio manager and trading platform.
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - day
+            - pool_id
+            - blockchain
+            - token_address
+            - token_symbol
+    columns:
+      - name: day
+        description: 'UTC event block time truncated to the day mark'
+      - name: pool_id
+      - name: pool_symbol
+        description: 'Symbol of the pool, consisting of the symbol of its tokens and their respective weights'
+      - name: blockchain 
+      - name: token_address
+      - name: token_symbol
```

Comment:
I think some of these columns have been declared before and you could inherit name, description, etc, using `*` mark